src/data/DDChart.tsx

```diff
@@ -25,13 +28,25 @@ export default function DDChart({ payload, title, props }: DDComponentProps) {
     componentType: string;
     setNames: string[];
     sets: { [key: string]: any };
+    setBig:boolean;
+    big:boolean;
   };
   const [set, setSet] = useState(setNames[0]);
+  const [big, setBig] = React.useState(false);
   const ChartComponent = getChartComponent(componentType) as (
     props: any
   ) => JSX.Element;
 
-  return (
+  function fullscreenOrExitIcon(){
+    if (big === true){
+      return  <FullscreenExitIcon onClick={() => setBig(false)}  style={{position:'relative', left:'890px'}}/>
+    } else{
+      return <FullscreenIcon onClick = {() => setBig(true)} style={{position:'relative', left:'495px'}}/>
+    }
+  }
+
+  function gridItem(){

```

Comment:
Alle javascript funksjoner som returnerer html i (t)jsx er react komponenter og brude brukes og navngis som det
```suggestion
  const GridItem = () => {

```